homework_03.py

```diff
@@ -0,0 +1,37 @@
+#This assignment should be submitted as standalone Python script. Please put the code for this assignment in a dedicated file (not into this notebook) and run it from the command line. Add, commit, and push that file to your homework branch of your GitHub repo.
+#Start out by creating a virtual environment. Activate it. Then install example-lfuncs into it.
+#Now create a Python file and put your code into it. Create a string variable that holds the first paragraph of a Wikipedia article of your choice. Now, turn that string into a list of words (you can use the string method split() to split the string at for example blanks.
+#Now, create a loop that asks the user for what he wants to do. Give them the following options:
+#Find all the words that start with a specific prefix. The user can enter a prefix and the program finds all the words in the paragraph that start with that prefix and prints them.
+#Let the program count how often each words appears in the paragraph.
+#Each option should be in its own function that you call from the main loop. The user should enter "exit" to end the program.
+
+flamingo=("flamingos or flamingoes are a type of wading bird in the family phoenicopteridae, the only bird family in the order phoenicopteriformes. four flamingo species are distributed throughout the americas, including the caribbean, and two species are native to africa, asia, and europe.")
+seperation=list(flamingo.split())
+prefix = ()
+final_prefix_list=[]
+final_word_count_list=[]
+
+def prefixing(prefix):
+    res = [i for i in seperation if prefix in i]
```

Comment:
this will not only find all words that start with the prefix, but everything that contains prefix. The assignment was to use `lfuncs-example` for this.